cfgov/unprocessed/apps/mmt-my-money-calendar/js/stores/input-wizard-store.js

```diff
@@ -0,0 +1,173 @@
+import { observable, action, computed, toJS } from 'mobx';
```

Comment:
I think we should pair on this store and heavily refactor it. There's a lot here that I would change, and I'm not sure I can describe verbally everything I'd do.